ddl/ddl_api.go

```diff
@@ -3996,6 +3996,7 @@ func (d *ddl) ExchangeTablePartition(ctx sessionctx.Context, ident ast.Ident, sp
 		Type:       model.ActionExchangeTablePartition,
 		BinlogInfo: &model.HistoryInfo{},
 		Args:       []interface{}{defID, ptSchema.ID, ptMeta.ID, partName, spec.WithValidation},
+		CtxVars:    []interface{}{[]int64{ntSchema.ID, ptSchema.ID}, []int64{ntMeta.ID, ptMeta.ID}},
```

Comment:
Why is it stored in `CtxVars`? I think we can decode the `Args` on demand.

Reply:
yes, we can get them from `Args`, but the order is different for different ddl job types, we have to calculate them very carefully and hard code them and make sure they will not be changed in the future, so I add them to `CtxVars` at first and second.

actually, I prefer to add to fields `RefTables` and `RefSchemas`, both ok for me. But I will not use `Args` to get them